### PR TITLE
html200 lesson1 assignment

### PR DESCRIPTION
**Pull request by kzvm**
This is the work i did for lesson1 assignment. I know I need to work on my style issues, but i hope I can fix the style issues later on and make the pages look better.

### CODE REVIEW

File: 1st-page/Blog.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    <meta charset="utf-8" />
+    <title>Page3 Lesson1 Assignment</title>
+</head>
+
+<style>
+
+    .header {
+        color: white;
+        background-color: #003366;
+        padding: 20px;
```
Comment (MadEste) [inline 1st-page/Blog.html]: This is fine for Assignment 2 but make sure up update css to use responsive units such as %, em, rem by lesson 3. px are no longer acceptable units (they are not responsive).

Comment (MadEste) [reply 1st-page/Blog.html]: These should really be updated to % or em by Assignment 3.

File: 1st-page/Blog.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        background-image: url("lesson04-mock2-banner-image.jpg");
+        background-repeat: no-repeat;
+        background-size: 1500px 160px;
```
Comment (MadEste) [inline 1st-page/Blog.html]: - No px units.

File: 1st-page/Blog.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    }
+
+    h3 {
+        color: #4CAF50;
+    }
+
+    .hidden {
+        position: relative;
+        left: -10000px;
```
Comment (MadEste) [inline 1st-page/Blog.html]: Using px to position elements is not good responsive design.

File: 1st-page/Blog.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+        top: auto;
+    }
+
+        .hidden:focus {
+            left: 0;
+        }
+
+    nav {
+        background-color: #DCDCDC;
+        padding: 30px;
```
Comment (MadEste) [inline 1st-page/Blog.html]: no px units

File: 1st-page/Blog.html [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    }
+
+    .travel img {
+        float: left;
+        margin-right: 40px;
+        margin-left: 40px;
```
Comment (MadEste) [inline 1st-page/Blog.html]: No px units

File: 1st-page/Blog.html [hunk 6]
```diff
@@ -0,0 +1,164 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Page3 Lesson1 Assignment</title>
+</head>
+
+<style>
```
Comment (MadEste) [inline 1st-page/Blog.html:9]: I would advise you move these stylesheets over to a dedicated css file applied to all pages.

File: 1st-page/styles/stylesheet.css [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+h3 {
+    width: 500px;
+    background-color: cadetblue;
+}
+
+.surfing {
+    font-family: 'Luckiest Guy', serif;
+    font-size: 75px;
```
Comment (MadEste) [inline 1st-page/styles/stylesheet.css]: No px units

File: 1st-page/index.html [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+        background-image: url("lesson04-mock4-footer-image.jpg");
+    }
+</style>
+
+<body>
+
+    <header>
+
+        <section class="header">
+            <img src="dog-logo.png" alt="dog logo">
```
Comment (MadEste) [inline 1st-page/index.html]: img src is not loading. (file is .PNG in caps)

File: 1st-page/index.html [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+            <a href="Blog.html">Blog</a> /
+            <a href="Checkout.html">Checkout</a>
+        </nav>
+
+    </header>
+
+    <section class="adoptIntro">
+
+        <img src="home-intro.jpg" alt="Owner hugging golden retrirver"
+             style="height:250px;width:300px;">
```
Comment (MadEste) [inline 1st-page/index.html:74]: Inline styles are generally bad practice and should be avoided. Also px are not responsive.

**Comment by kzvm** [comment]
When I made the change using the conversion calculator from px to em,  made it worst. I will try to fix that so it looks the same as it looks right now with the px.

Thanks.

**Comment by kzvm** [comment]
Thanks, I never found why it was not working, did not think about lower case or capital letters. That’s awesome I will get that fix thanks again.

Kerwin

**Comment by kzvm** [comment]
Hello Estevan,

I have a hard time targeting individual images to style them. I can use some help on targeting individual images when there are more than one image on the code.

Thanks

File: 1st-page/DogsPage.html [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+            <h4>Lucas</h4>
+            <p>Cost to Adopt: $123.45</p>
+            <p>
+                Corrum volorit iandae<br />
+                nimaxim cum restia volor<br />
+                reicid ut et etur sunt arum<br />
+                rendae pla endis re ea erum<br />
+                qui doluptae
+            </p>
+            <button onclick="onDogPrice(123)" value="123" id="luc">Adopt</button>
```
Comment (MadEste) [inline 1st-page/DogsPage.html:141]: Good job passing parameters.

File: 1st-page/dogs.js [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+    let luna = alert('Luna, Adult breed, $123');
+}
+
+function stellaImg() {
+    let stella = alert('Stella, Adult breed, $123');
+}
+
+let total = 0;
+
+function onDogPrice(price) {
```
Comment (MadEste) [inline 1st-page/dogs.js:55]: Good work update the price function.

File: 1st-page/dogs.js [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+//function murphyImg() {
+//    let murphy = alert('Murphy, Adult breed, $123');
+//}
+
+//function violetImg() {
+//    let violet = alert('Violet, puppy breed, $123');
+//}
+
+function piperImg() {
```
Comment (MadEste) [inline 1st-page/dogs.js:13]: While this approach works, it is inefficient. A better approach would be to have one function for all dog image clicks, and instead pass it parameters to alert. Similar to the approach for the onDogPrice() function below.

File: 1st-page/index.html [hunk 13]
```diff
@@ -0,0 +1,193 @@
...
+            <a href="index.html">Home</a> /
+            <a href="DogsPage.html">Dogs</a> /
+            <a href="Blog.html">Blog</a> /
+            <a href="Checkout.html">Checkout</a>
+        </nav>
+
+    </header>
+
+    <section class="preview">
+        <a href="Blog.html"><img src="blogPreview.png" alt="Blog Preview"
```
Comment (MadEste) [inline 1st-page/index.html]: This should be html code and not an image.

File: 1st-page/styles/stylesheet.css [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.preview img {
+    float: right;
+    margin-right: 50px;
+}
+
+@media screen and (max-width: 768px) and (orientation: portrait) {
+    .preview {
+        position: absolute;
```
Comment (MadEste) [inline 1st-page/styles/stylesheet.css]: You should not be hiding this using position: absolute. Try toggling visibility instead.

File: 1st-page/styles/stylesheet.css [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+        margin-top: 100px;
+        margin-right: 30px;
+        width: 15%;
+        border: 3px solid #73AD21;
+        padding: 15px;
+    }
+
+@media only screen and (max-width: 576px) {
+    .flex {
+        display: inline;
```
Comment (MadEste) [inline 1st-page/styles/stylesheet.css:83]: should be using grid-template-columns: auto auto;

**Pull request closed without merging**